examples/tutorials/Putting_Multitask_Learning_to_Work.ipynb

```diff
@@ -8,7 +8,7 @@
    },
```

Comment:
<p>@The formatting here appears wrong. Can you please take another look?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>